src/Components/UpdateProfilePhotoForm.php

```diff
@@ -35,9 +35,14 @@ public function updatedImageSingle(): void
     {
         $this->validateImageSingle();
 
-        $file = $this->imageSingle->storePubliclyAs('uploads', $this->imageSingle->hashName());
+        $file = $this->imageSingle;
```

Comment:
Since `imageSingle` is an instance of `TemporaryUploadedFile` it's already stored in a temp folder. Here we get its real path and pass it to media-library so it can be moved to S3.